colcon-labeled-test/action.yaml

```diff
@@ -0,0 +1,118 @@
+name: colcon-labeled-test
```

Comment:
Why do you not extend `colcon-build-and-test`?
I think adding an argument can also support this use case.
Could you show me the difference?

Reply:
> Why do you not extend colcon-build-and-test?

First, I extended `colcon-build-and-test`. However, this is complicated and the test jobs are only executed sequentially. So, I created a new workflow. This will run in parallel, so we can scale the test jobs.

My old workflow is here:
https://github.com/KeisukeShima/autoware-github-actions/blob/0e8e8df91e75621659acb567edd89706c556fd35/colcon-build-and-test/action.yaml#L79-L136

Reply:
I see, it's reasonable.

But in that case, don't you have to remove `colcon test` from `colcon-build-and-test`?
I'm concerned about whether the test runs twice.

Reply:
Yes, I'm considering modifying the workflow for `colcon-build-and-test`.
My idea is to create a `skip-test` as an input to `colcon-build-and-test`. But that would skip all tests including lint, copyright, etc. That is a problem.
Do you have any ideas to solve this?

Reply:
I believe preparing `colcon-build` and `colcon-test` is good.
And in the `colcon-test` workflow, add the `label` support.

It can reduce duplication.

Reply:
Sounds good! I will modify workflow.

Reply:
@KeisukeShima Ah, but we have to consider the compatibility.
So could you create two new workflows from `colcon-build-and-test`, but please don't remove `colcon-build-and-test`.

Reply:
We'll just mark `colcon-build-and-test` as `deprecated`.

Reply:
Yes. We already use `colcon-build-and-test` workflow in many repos. So I need to change it carefully.

Reply:
Thank you for your work!
Could you rename the title?

Reply:
Added new build workflow
746b4da

And modify test workflow
99a68a1

Reply:
Since this is the first commit for adding actions(features), I removed the scope.